### ISSUE TITLE
flash chess how to play a rochade

### ISSUE THREAD

**Issue by t-web007** [baranusluel/square-on#1]

**Comment by baranusluel** [comment]
Good point! Castling is not yet supported, but it could be implemented by first moving one of the pieces out of the way and then moving the other along the edges of the squares, similar to the capturing mechanism.

While the project is no longer under development, anyone is welcome to submit a PR with the changes.
